docs/03-extensions/stripe/reference/environment-variables.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Environment variables"
+description:
+  "This reference lists environment variables used by the Stripe extension"
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+:::info
+
+Your API keys can be found on your
+[Stripe Dashboard](https://dashboard.stripe.com/account/apikeys)
+
+:::
+
+| Name                                    | Description                                                                                                                                     | Default value |
+| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| FRONT_COMMERCE_STRIPE_SECRET_KEY        | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |
+| FRONT_COMMERCE_STRIPE_PUBLISHABLE_KEY   | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |
+| FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY | Max network retry to Stripe network before consider call as a failure                                                                           | `3`           |
```

Comment:
**thought:** [In the v2 documentation](https://developers.front-commerce.com/docs/2.x/advanced/payments/stripe#advanced-network-retries), this environment variable deserved a whole section.

**question:** Is there a reason that isn't the case anymore?

Reply:
There's no acceptance criterias in the inital issue and i think it speak by itself. I've added a link to the stripe documentation.